src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -143,19 +143,19 @@ struct npc_doomfire_spirit : public ScriptedAI
 {
     npc_doomfire_spirit(Creature* creature) : ScriptedAI(creature){ }
 
-    float const m_turnConstant = 0.785402f;
-    float m_fAngle = urand(0, M_PI * 2);
+    float const _turnConstant = 0.785402f;
+    float _fAngle = urand(0, M_PI * 2);
```

Comment:
```suggestion
    float const TurnConstant = 0.785402f;
    float FAngle = urand(0, M_PI * 2);
```

Only private members use _ prefix
https://www.azerothcore.org/wiki/cpp-code-standards#standard-prefixes-for-constants